bulbs/feeds/views.py

```diff
@@ -40,3 +43,21 @@ def get_context_data(self, *args, **kwargs):
             content.feed_url = self.request.build_absolute_uri(feed_path)
 
         return RequestContext(self.request, context)
+
+
+class SpecialCoverageRSSView(RSSView):
+    """Really simply, this syndicates Content."""
+    feed_title = "Special Coverage RSS Feed"
+
+    def get_queryset(self):
+        sc_id = self.request.GET.get("special_coverage_id")
+        sc_slug = self.request.GET.get("special_coverage_slug")
+
+        if sc_id:
+            sc_ids = SpecialCoverage.objects.get(id=sc_id).query["included_ids"]
+        elif sc_slug:
+            sc_ids = SpecialCoverage.objects.get(slug=sc_slug).query["included_ids"]
+        else:
+            sc_ids = []
```

Comment:
I think we actually want to deliver an empty RSS in this scenario.  If they provide a slug or ID that doesn't map to anything, it should just be empty